frontend/src/pages/talks/TalkSingleEntry.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { Box, Paper, Typography } from '@mui/material';
+import { TalkEntry } from 'src/services/mocks';
+import { extractVideoId } from 'src/utils/video';
+import { useTranslation } from 'react-i18next';
+
+const TalkSingleEntry = ({ talk }: { talk: TalkEntry; compact?: boolean }) => {
+  const { i18n } = useTranslation();
+
+  const abstractParagraphs = talk.abstract.split('\n');
+
+  function isPast(talk: TalkEntry) {
+    return new Date(talk.date) < new Date();
+  }
+
+  const title = (
+    <Box display="flex" gap={2}>
+      <Typography variant="h4">{talk.title}</Typography>
+    </Box>
+  );
+
+  const formatDate = (dateString: string) => {
+    if (i18n.language == 'fr') {
+      const date = new Date(dateString);
+
+      const day = date.getDate();
+      const month = date.getMonth() + 1;
+      const year = date.getFullYear();
+
+      const formattedDate = `${day < 10 ? '0' + day : day}/${
+        month < 10 ? '0' + month : month
+      }/${year}`;
+
+      return formattedDate;
+    }
+    return dateString;
+  };
+
+  return (
+    <>
+      <a
+        className="no-decoration"
+        href={isPast(talk) ? talk.youtube_link : talk.invitation_link}
+      >
+        <Paper sx={{ mb: 2 }}>
```

Comment:
There is already a margin bottom defined in the `TalkEntryList` for each element, I think we can freely remove this `sx`.